diff --git a/node_modules/@workflow/world-vercel/dist/utils.js b/node_modules/@workflow/world-vercel/dist/utils.js
index 5df338c..4a7e623 100644
--- a/node_modules/@workflow/world-vercel/dist/utils.js
+++ b/node_modules/@workflow/world-vercel/dist/utils.js
@@ -117,6 +117,7 @@ export async function makeRequest({ endpoint, options = {}, config = {}, schema,
     const response = await fetch(url, {
         ...options,
         headers,
+        cache: 'no-store'
     });
     if (!response.ok) {
         const errorData = (await response.json().catch(() => ({})));
@@ -131,6 +132,7 @@ export async function makeRequest({ endpoint, options = {}, config = {}, schema,
     }
     try {
         const text = await response.text();
+        console.log(text);
         return schema.parse(JSON.parse(text));
     }
     catch (error) {
